core/src/main/java/org/mule/api/debug/Debuggable.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.api.debug;
```

Comment:
Is API as many different projects will depend on it (mule, debugger, devkit, extensions API)